arangod/Agency/AgentCallback.cpp

```diff
@@ -42,6 +42,7 @@ bool AgentCallback::operator()(arangodb::ClusterCommResult* res) {
   if (res->status == CL_COMM_SENT) {
     if (_agent) {
       auto body = res->result->getBodyVelocyPack();
+      auto success = body->slice().get("success").isTrue();
```

Comment:
success can also be false if "success" is missing from the message.  Is the current path for !success still valid, or should there be a logical path for "bad message encoding"?

Reply:
Good.